manifest-version.py

```diff
@@ -0,0 +1,20 @@
+""" Script to change versioning of manifest.yml for executors
+[encoders, crafters, indexers, rankers]. Invoke this script from the
+executor directory passing old version and new version.
+Example usage from TextPaddleHubEncoder executor directory:
+python ../../../manifest-version.py
+"""
+import os, fnmatch
+old_version = "version: 0.0.1"
+latest_version = "version: 0.0.2"
+filePattern = "manifest.yml"
```

Comment:
I would against using a hardcoded replacement for bumping version.

Reply:
yes sure, removed all hardcoding